Scenarios Objectives -- ATHZ - INPV (#609)
OWASP/wstg: document/4-Web_Application_Security_Testing/05-Authorization_Testing/03-Testing_for_Privilege_Escalation.md

Usually, people refer to *vertical escalation* when it is possible to access resources granted to more privileged accounts (e.g., acquiring administrative privileges for the application), and to *horizontal escalation* when it is possible to access resources granted to a similarly configured account (e.g., in an online banking application, accessing information related to a different user).

## Test Objectives

> - Identify injection point related to privilege manipulation.
> - Fuzz and try to bypass possible security measures in place.

Reviewer: ```suggestion
- Identify injection points related to privilege manipulation.
- Fuzz or otherwise attempt to bypass security measures.
```
